assets/js/src/config.js

```diff
@@ -1,2 +1,36 @@
 /* global wcgai */
 export const config = wcgai.config;
+export const EEARegions = [
+	'AT',
+	'BE',
+	'BG',
+	'HR',
+	'CY',
+	'CZ',
+	'DK',
+	'EE',
+	'FI',
+	'FR',
+	'DE',
+	'GR',
+	'HU',
+	'IS',
+	'IE',
+	'IT',
+	'LV',
+	'LI',
+	'LT',
+	'LU',
+	'MT',
+	'NL',
+	'NO',
+	'PL',
+	'PT',
+	'RO',
+	'SK',
+	'SI',
+	'ES',
+	'SE',
+	'UK',
```

Comment:
I know this is based on the snippet Google provided but `UK` doesn't appear to be a valid country code. Testing the branch as it, there is no `On-page default` set:

<img width="827" alt="Screenshot 2024-03-01 at 12 46 17" src="https://github.com/woocommerce/woocommerce-google-analytics-integration/assets/40762232/8c5bf742-a679-495e-8411-3fe7d4548b9a">

Updating it to `GB` fixes that:

<img width="790" alt="Screenshot 2024-03-01 at 12 46 44" src="https://github.com/woocommerce/woocommerce-google-analytics-integration/assets/40762232/f7cc9d79-1c32-46bc-a34e-6b88e9255bf4">

```suggestion
	'GB',
```